File: atcoder/abc321a_321_like_checker.hs
-- vicfred
-- https://atcoder.jp/contests/abc321/tasks/abc321_a
-- implementation
import Control.Applicative (liftA2)
import Data.Char (digitToInt)
import Data.Bool (bool)

main :: IO ()
main = map digitToInt <$> getLine >>= putStrLn .bool "No" "Yes"
                                               .all (== True)
                                               .liftA2 (zipWith (>)) id tail

File: atcoder/abc321A_321_like_checker.py
# vicfred
# https://atcoder.jp/contests/abc321/tasks/abc321_a
# implementation
N = list(input())
flag = True
for i in range(len(N) - 1):
    if int(N[i]) <= int(N[i+1]):
        flag = False
if flag:
    print("Yes")
else:
    print("No")
